core/helpers/EEH_MSG_Template.helper.php

```diff
@@ -298,7 +298,7 @@ public static function get_shortcodes(
                     $_field = $field;
                 }
                 if (isset($_field)) {
-                    $localized[ $_field ] = $shortcodes;
+                    $localized[ (string) $_field ] = $shortcodes;
```

Comment:
gosh we repeat the seemingly fragile pattern I noted above elsewhere too :)  I'm looking at this whole code block and shuddering lol.  Refactoring is outside the scope of this issue though so for now I guess it'll have to do.

Reply:
🙈